lib/tapjoy/ldap/base.rb

```diff
@@ -181,6 +184,30 @@ def set_id_boundary(role)
 
         return minID, maxID
       end
+
+      # Load config from files
+      def load_config_from_files(ldap_config_file, ldap_password_file)
+        ldap_info     = YAML.load_file(ldap_config_file)
+        ldap_password = File.read(ldap_password_file).chomp
+        @rootdn       = ldap_info['rootdn']
+        @hosts        = ldap_info['servers']
+        @basedn       = ldap_info['basedn']
+        @conn         = find_valid_host(ldap_password)
+        @service_ou   = ldap_info['service_ou']
+        @email_domain = ldap_info['email_domain']
+        @port         = ldap_info['port']
+      end
+
+      # Load config from ENV
+      def load_config_from_env
+        @rootdn       = ENV['LDAP_BIND_DN']
+        @hosts        = Array(ENV['LDAP_SERVERS'])
```

Comment:
I may be mistaken, but this does not appear to function in the intended fashion using common array separators:

```
[7] pry(main)> this = "stuff,junk"
=> "stuff,junk"
[8] pry(main)> Array(this)
=> ["stuff,junk"]
[9] pry(main)> this = "stuff junk"
=> "stuff junk"
[10] pry(main)> Array(this)
=> ["stuff junk"]
```

Reply:
i was actually asking about this on a #slack channel; doesn't seem intuitive on how one would use this.  should at least have some documentation??

Reply:
So, the goal of this is to make a single string into an array or leave an array as-is

```
irb(main):005:0> test=%w(stuff junk)
=> ["stuff", "junk"]
irb(main):006:0> Array(test)
=> ["stuff", "junk"]
irb(main):007:0> test='stuff junk'
=> "stuff junk"
irb(main):008:0> test
=> "stuff junk"
```

Reply:
oh, I see the problem.  Oops